src/core/database/database.service.ts

```diff
@@ -302,14 +306,22 @@ export class DatabaseService {
           active: true,
           createdAt,
         }),
-        node('newPropNode', 'Property', newPropertyNodeProps),
+        node('newPropNode', propLabels, newPropertyNodeProps),
```

Comment:
Should I use this instead of grabbing node labels dynamically in the query to mark as `Deleted_`?

Reply:
Up to you. This certainly seems easier... Do we need to work about an additional Deleted_ prefix as we have in other places?

Reply:
You're asking if we should worry about prefixing properties on update?

Reply:
Sems like deletion and updating are the same because we never truly delete things. I suppose we could label with `Updated_` property and an `updatedAt` property on the old prop.

Reply:
Nah I wouldn't introduce another prefix. Stick with `Deleted_` label prefix and deletedAt property